### ISSUE TITLE
Преимущества Drupal -> Стабильность

### ISSUE THREAD

**Issue by duozersk** [dru-io/promo#17]
Этот пункт можно смело назвать не совсем верным, я бы даже сказал "лживым".
"Каждый модуль на Drupal.org проходит тщательную модерацию, что обеспечивает стабильную работу вашего сайта" - далеко не каждый полноценный модуль, не говоря уже о модулях песочницы. А с введением права одного полноценного модуля на аккаунт безо всякой проверки так вообще...

Предлагаю как минимум следующую корректировку:
"Множество популярных модулей на Drupal.org прошло тщательную модерацию, что гарантирует стабильную работу вашего сайта"

**Comment by Niklan** [comment]
Да, в текущих реалиях там вообще может быть что угодно, к сожалению. Предложенное описание очень удачно подойдет.

**Comment by duozersk** [comment]
Да и не только в текущих. В прежних тоже через круги ада проходил только один модуль на аккаунт разработчика, а дальше публикуй что хочешь.
